apps/web/middleware.ts

```diff
@@ -190,6 +188,7 @@ export const config = {
     "/future/video/:path*",
     "/teams",
     "/future/teams/",
+    "/settings/:path*",
```

Comment:
- we need to trigger middleware for all settings pages (all rendered by app router now)
- otherwise, there aren't custom values set in headers like `x-locale`.